<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QGroupBox" name="ResultGroup">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>530</y>
     <width>1001</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yahei Mono</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox 
{
	border:1px solid #0000EE;
	border-radius:5px;
	margin-top:1ex;
}
QGroupBox::title 
{
	subcontrol-origin: margin;
	subcontrol-position: top center;
	padding: 0 3px;
}</string>
   </property>
   <property name="title">
    <string>检测结果</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>831</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="Layout2_1">
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="6">
      <widget class="QLabel" name="Label2_3">
       <property name="font">
        <font>
         <family>Yahei Mono</family>
         <pointsize>26</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>总 记:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="Label2_2">
       <property name="font">
        <font>
         <family>Yahei Mono</family>
         <pointsize>26</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>次品数:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="Label2_1">
       <property name="font">
        <font>
         <family>Yahei Mono</family>
         <pointsize>26</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>正品数:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="Label2_zcps">
       <property name="font">
        <font>
         <family>Yahei Mono</family>
         <pointsize>26</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>8000</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer2_1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="7">
      <widget class="QLabel" name="Label2_zs">
       <property name="font">
        <font>
         <family>Yahei Mono</family>
         <pointsize>26</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>16000</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="Label2_zzps">
       <property name="font">
        <font>
         <family>Yahei Mono</family>
         <pointsize>26</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>8000</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <spacer name="horizontalSpacer2_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="Button1_ccxx">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>20</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>尺寸信息</string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Button1_slxx">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>60</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>数量信息</string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Button1_qtsz">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>100</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>其它设置</string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_AppTile">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yahei Mono</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 170, 170);</string>
   </property>
   <property name="text">
    <string>机器视觉智能终端</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="ImageGroup">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>651</width>
     <height>381</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yahei Mono</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox 
{
	border:1px solid #0000EE;
	border-radius:5px;
	margin-top:1ex;
}
QGroupBox::title 
{
	subcontrol-origin: margin;
	subcontrol-position: top center;
	padding: 0 3px;
}</string>
   </property>
   <property name="title">
    <string>纽扣图像</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="Label1_Image1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>192</width>
      <height>192</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 255, 127);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Label1_ResIcon1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>290</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../Res/Resource.qrc">:/icon/right_48px.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="Label1_Image2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>80</y>
      <width>192</width>
      <height>192</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 255, 127);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Label1_ResIcon2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>290</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image:url(:/icon/right_48px.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Label1_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>192</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yahei Mono</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>相机1 未连接</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Label1_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>192</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yahei Mono</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>相机2 未连接</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Label1_3">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>30</y>
      <width>192</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yahei Mono</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>相机3 未连接</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Label1_Image3">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>80</y>
      <width>192</width>
      <height>192</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 255, 127);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Label1_ResIcon3">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>290</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image:url(:/icon/right_48px.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="ControlGroup">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>170</y>
     <width>331</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yahei Mono</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox 
{
	border:1px solid #0000EE;
	border-radius:5px;
	margin-top:1ex;
}
QGroupBox::title 
{
	subcontrol-origin: margin;
	subcontrol-position: top center;
	padding: 0 3px;
}</string>
   </property>
   <property name="title">
    <string>检测控制</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QPushButton" name="WorkStartButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yahei Mono</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>检测</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="WorkStopButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yahei Mono</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(239, 41, 41);

</string>
    </property>
    <property name="text">
     <string>停止</string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="SelectGroup">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>70</y>
     <width>331</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yahei Mono</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox 
{
	border:1px solid #0000EE;
	border-radius:5px;
	margin-top:1ex;
}
QGroupBox::title 
{
	subcontrol-origin: margin;
	subcontrol-position: top center;
	padding: 0 3px;
}</string>
   </property>
   <property name="title">
    <string>纽扣选择</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QPushButton" name="ButtonSelectButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yahei Mono</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>纽扣选择</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_ButtonImage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/icon/button.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_ButtonId">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yahei Mono</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>S120231-001</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="StatusGroup">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>320</y>
     <width>331</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yahei Mono</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox 
{
	border:1px solid #0000EE;
	border-radius:5px;
	margin-top:1ex;
}
QGroupBox::title 
{
	subcontrol-origin: margin;
	subcontrol-position: top center;
	padding: 0 3px;
}</string>
   </property>
   <property name="title">
    <string>连接状态</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>681</y>
     <width>1001</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="Layout6_1">
    <property name="spacing">
     <number>15</number>
    </property>
    <item>
     <widget class="QPushButton" name="pushButton_Manage">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Yahei Mono</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	padding:10px;
	color:#FFFFFF;
	background:#34495E;
	border-style:none;
	border-radius:5px;
}
QPushButton:hover
{	
	color:#F0F0F0;
	background:#4E6D8C;
}
QPushButton:pressed
{
	color:#B8C6D1;
	background:#2D3E50;
}</string>
      </property>
      <property name="text">
       <string>文件管理</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_Learn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Yahei Mono</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	padding:10px;
	color:#FFFFFF;
	background:#34495E;
	border-style:none;
	border-radius:5px;
}
QPushButton:hover
{	
	color:#F0F0F0;
	background:#4E6D8C;
}
QPushButton:pressed
{
	color:#B8C6D1;
	background:#2D3E50;
}</string>
      </property>
      <property name="text">
       <string>机器学习</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_Statistics">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Yahei Mono</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	padding:10px;
	color:#FFFFFF;
	background:#34495E;
	border-style:none;
	border-radius:5px;
}
QPushButton:hover
{	
	color:#F0F0F0;
	background:#4E6D8C;
}
QPushButton:pressed
{
	color:#B8C6D1;
	background:#2D3E50;
}</string>
      </property>
      <property name="text">
       <string>统计分析</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_Settings">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Yahei Mono</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	padding:10px;
	color:#FFFFFF;
	background:#34495E;
	border-style:none;
	border-radius:5px;
}
QPushButton:hover
{	
	color:#F0F0F0;
	background:#4E6D8C;
}
QPushButton:pressed
{
	color:#B8C6D1;
	background:#2D3E50;
}</string>
      </property>
      <property name="text">
       <string>系统设置</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_Help">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Yahei Mono</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	padding:10px;
	color:#FFFFFF;
	background:#34495E;
	border-style:none;
	border-radius:5px;
}
QPushButton:hover
{	
	color:#F0F0F0;
	background:#4E6D8C;
}
QPushButton:pressed
{
	color:#B8C6D1;
	background:#2D3E50;
}</string>
      </property>
      <property name="text">
       <string>帮助</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_AppClose">
   <property name="geometry">
    <rect>
     <x>967</x>
     <y>8</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	border-style:none;
	background-image: url(:/icon/error_48px.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../Res/Resource.qrc"/>
 </resources>
 <connections/>
</ui>
